test/integration/targets/azure_rm_devtestlab/tasks/main.yml

```diff
@@ -173,51 +173,69 @@
       - output.virtualnetworks[0]['description'] != None
       - output.virtualnetworks[0]['provisioning_state'] != None
 
-#- name: Create instance of DevTest Labs artifacts source
-#  azure_rm_devtestlabartifactsource:
-#    resource_group: "{{ resource_group }}"
-#    lab_name: "{{ lab_name }}"
-#    name: "{{ artifacts_name }}"
-#    uri: https://github.com/Azure/azure_preview_modules.git
-#    source_type: github
-#    folder_path: /tasks
-#    security_token: "{{ github_token }}"
-#  register: output 
-#- name: Assert if the change was correctly reported
-#  assert:
-#    that:
-#      - output.changed
+- name: Create instance of DevTest Labs artifacts source
+  azure_rm_devtestlabartifactsource:
+    resource_group: "{{ resource_group }}"
+    lab_name: "{{ lab_name }}"
+    name: "{{ artifacts_name }}"
+    uri: https://github.com/Azure/azure_preview_modules.git
+    source_type: github
+    folder_path: /tasks
+    security_token: "{{ github_token }}"
+  register: output 
+  when: "github_token | length > 0"
+- name: Assert if the change was correctly reported
+  assert:
+    that:
+      - output.changed
+  when: "github_token | length > 0"
```

Comment:
this is  always false, right? when it could be true?

Reply:
in azure_preview_modules will be true
or if somebody adds GitHub_token in the env

Reply:
I am testing the same change in azure_preview_modules as well.
the idea is that we have the same test in both.